wordpress-mobile/WordPress-Android
# Issue/update format icons

### Fix
Update the editor format button active/highlighted state color to `wp_blue` (`#0087be`).

### Test
0. Enable ***Beta*** editor.
1. Go to ***Sites*** tab.
2. Tap ***Create*** floating button.
3. Tap content text field.
4. Tap format toolbar button.
5. Notice updated format toolbar button color.
